src/Tizen.NUI/src/public/Layouting/LayoutItem.cs

```diff
@@ -308,11 +308,11 @@ public void RequestLayout()
             Flags = Flags | LayoutFlags.ForceLayout;
             if (Parent != null)
             {
-                 LayoutGroup layoutGroup =  Parent as LayoutGroup;
-                 if(! layoutGroup.LayoutRequested)
-                 {
-                    layoutGroup.RequestLayout();
-                 }
+                LayoutGroup layoutGroup =  Parent as LayoutGroup;
+                if(! layoutGroup.LayoutRequested)
```

Comment:
follow C# code format -> (Ctrl+K,F in Visual studio)

Reply:
Thanks I did.